scripts/install_deps.sh

```diff
@@ -1,6 +1,27 @@
 #!/bin/bash
-
 apt-get update
 apt-get update --fix-missing
-DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get -y install tzdata
-apt-get -y install zip unzip libncurses5 wget git build-essential cmake curl libgmp-dev libssl-dev libzstd-dev time zlib1g-dev libtinfo-dev bzip2 libbz2-dev python3 python3-numpy file
+DEBIAN_FRONTEND='noninteractive'
+TZ='Etc/UTC'
```

Comment:
This won't work. Either set them before the call or `export` them.

Reply:
Done.

Reply:
> This won't work. Either set them before the call or `export` them

Actually, this would work. The variable values would change during the script, they just wouldn't be exported to child processes.

Reply:
Pedantically: the mode would remain interactive and the user would be queried for timezone.